_episodes/02-markdown.md

```diff
@@ -31,10 +31,54 @@ keypoints:
 - put directory under version control with git
   - link to SWC git-novice
 
+# Markdown
+Markdown is a lightweight markup language, i.e. a convention for adding
+style information to textual content.
+As the name Markdown indicates, the syntax elements of this language
+are shut *down* to a minimum.
+Having a rather minimalistic syntax, text formatted in Markdown is comparably
+well readable.
+This might be one reason for Markdown having become the language of choice
+for formated user input on websites like for example
+- [Stack Exchange](https://stackexchange.com/)
+- [GitHub](https://github.com/)
+- [GitLab](https://about.gitlab.com/).
+
+
+# Where to start writing Markdown?
+A lot of tools for rendering Markdown source code exist.
+Rendering is the process of generating a nice view of the content
+using the style information included in the source text.
+Chances are high, your editor can do this.
+As we are working towards authoring websites using Jeckyll and GitHub pages,
+we will use GitHub straight away for learning the basics of Markdown.
+The GitHub project you created in the last episode contains a file `README.md`.
+
+The picture below shows the projects default view.
+This view includes a rendered view of the content inside the file `README.md`.
+Your project should look quite similar except for the red circle around the pencil symbol.
+
+![Warning: You should see a picture here displaying: Github repository for the group website](../fig/group_website_repo_edit.png)
+
+You can click on that pencil symbol to open an edit view of your projects README file.
+This looks like
```

Comment:
Maybe want to add what it looks like in word form here, in case folks can't see the image.

```suggestion
Once you've clicked the edit button, GitHub will open that file in its web text editor.
```

Reply:
I'll reuse the wording `editing interface` and your suggestion!